apps/SageAccountWeb/package.json

```diff
@@ -72,6 +72,7 @@
   "scripts": {
     "clean": "rimraf build coverage",
     "start": "vite",
+    "preview": "vite preview",
```

Comment:
Shortcut to locally serve the built output (so I can just run `nx run SageAccountWeb:preview --port 3000` before running SWC e2e tests locally)